udata/templates/search.html

```diff
@@ -134,7 +138,15 @@
 {% block main_content %}
 <div class="tab-content">
 
-    <div class="tab-pane row active" id="datasets">
+    {% if not datasets and not reuses and not organizations and not users and not territories %}
```

Comment:
Not sure, it's mentioned on the [wikipedia page](https://en.wikipedia.org/wiki/Typographical_error)